NewEvaluation.css

```diff
@@ -180,6 +148,52 @@ h3{
 
 }
 
+/*---------------------- TextArea Box---------------------------*/
+
+.user-textarea textarea{
+	width:100%;
+	height: 100px;
+}
+
+
+
+/*------------------- Legend Box ------------------*/
+.legend-box legend{
+	color:#03a8f3;
+	font-size: 18px;
+	font-weight: 500;
+	padding-left: 1em;
+	padding-right: 1em;
+}
+
+.legend-box li{
+	float:left;
+	width: 33%;
+	padding-bottom: 10px;
+}
+.legend-box select{
+	width: 40%;
+	padding: 3px;
+}
+.legend-box label{
+	display: inline-block;
+	width: 49%;
+}
+
+.legend-box{
```

Comment:
I believe Sebi had a comment on another PR, stating that it helps better understand the structure and behavior of your stylesheets if you order your selectors in their natural order of appearance in the DOM. So, this particular ruleset might be better placed just below L160. Optional, since this will still work, but a good practice which improves legibility, nonetheless.